pkg/util/ready/ready.go

```diff
@@ -100,6 +102,24 @@ func CheckDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInte
 	}
 }
 
+// CheckAROOperatorDeploymentIsReady returns a function which polls AROOperator Deployment and
+// returns its readiness
+func CheckAROOperatorDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInterface, name string) (bool, error) {
```

Comment:
Not clear from the name and/or comment that this function checks oprator version.